test/puppet_processor_test.rb

```diff
@@ -21,4 +21,110 @@ def test_keys_are_strings
       assert_snapshot(result, snapshot_filename)
     end
   end
+  
+  def test_deb
```

Comment:
This was removed as part of https://github.com/theforeman/smart_proxy_host_reports/commit/ca6d93e2ac651af29ad38205d1eca6accd7a3313. I guess there is no need to return them.

I guess all the changes here should not be in this PR.